Description: Fix FTBFS with GCJ-4.4:
 "Cannot override the final method from ResourceBundle"
 Disable function clearCache() which is not used anyway.
Bug-Debian: http://bugs.debian.org/531995
Author: Torsten Werner <twerner@debian.org>
Forwarded: no
Last-Update: 2010-03-20
--- a/src/org/apache/axis/i18n/ProjectResourceBundle.java
+++ b/src/org/apache/axis/i18n/ProjectResourceBundle.java
@@ -360,10 +360,12 @@
     /**
      * Clears the internal cache
      */
+    /*
     public static void clearCache()
     {
         bundleCache.clear();
     }
+    */
     
     public String toString() {
         return resourceName;
